masterbase/app.py

```diff
@@ -165,6 +166,20 @@ async def demodata(request: Request, api_key: str, session_id: str) -> Redirect:
     )
 
 
+@get("/db_export", guards=[valid_key_guard, analyst_guard], sync_to_thread=False)
+def db_export(request: Request, api_key: str, data: DBExportBody) -> Stream:
+    """Return a database export of the requested `table` from within the last `max_age` seconds."""
+    engine = request.app.state.engine
+    filename = f"demo_sessions-{datetime.now()}.csv"
+    return Stream(
+        db_export_cached(engine, data.table, data.max_age),
+        headers={
+            "Content-Type": "text/csv",
+            "Content-Disposition": f"attachment; filename={filename}",
+        },
+    )
+
+
```

Comment:
would be nice to add a test for this given we have the infra

Reply:
how mock demo_sessions data

Reply:
i didn't bother, used the reports table instead